wikimedia/wikipedia-ios
# Update fix-reverse-dollar-signs.sh

Phabricator: N/A

### Notes:
I Used `find -print0` and` xargs -0` instead of `find -print` and `xargs -L 1` allowing to handle filenames with spaces or special characters more safely & I also removed the `-L 1` option from the xargs command, as this is not necessary when using 
`-print0` and` -0 `together.

### Test Steps:

1. Open a terminal or command prompt in the directory where the `'.strings'` files are located.
2. Type the script and press Enter to execute it
3. The script should run and perform the substitutions on all the matching `'.strings'` files
4. You can verify that the script worked by opening the `'.strings' `files and checking if the substitutions have been made.
